check_addon.py

```diff
@@ -103,6 +107,7 @@ def _check_for_invalid_xml_files(error_counter, file_index):
                 # Just try if we can successfully parse it
                 xml.etree.ElementTree.parse(xml_path)
             except xml.etree.ElementTree.ParseError:
+                xml_path = relative_path(xml_path)
```

Comment:
variable shouldn't be overwritten, either create a new variable, or do the `relative_path` function call directly in the log call